src/types/contract.ts

```diff
@@ -1,10 +1,12 @@
 import { NetworkId } from './networkId';
 import { TUuid } from './uuid';
+import { TAddress } from './address';
 
 export interface Contract {
   name: string;
+  label?: string; // Added in InteractWithContracts/stateFactory.. !?
```

Comment:
Can we remove this in favor of `name` then?

Reply:
I only found where it was set but not where it was called and wanted to avoid touching/breaking something that isn't tested yet.

Reply:
Okay, let's make sure it's removed soon though. Can we add a TODO?